integration/setup/setup.go

```diff
@@ -38,14 +38,27 @@ func WrapTestMain(g *framework.Guest, h *framework.Host, m *testing.M) {
 
 	err := Setup(g, h)
 	if err != nil {
-		log.Printf("%#v\n", err)
+		log.Printf("\nSetup stage failed,")
```

Comment:
These newlines look a bit all over the place to me, what does this output look like?

Reply:
extra new lines for some separation its mostly intended that each stage has atleast  1 empty line between each other and also including the separator so it should look like this:
```
<... setup stage output ... >
<empty line>
Setup stage finished/failed
<empty line>
< .... test stage output ... >
<empty line>
Test stage finished/failed
<empty line>
< ... teardown stage output ... >
<empty line>
Teardown stage finished/failed
```

Reply:
It makes sense to me, do you think its bad idea?

Reply:
Having the missing error handling makes sense. I am not sure about the logging though. This should actually be aligned with other projects and we should also use our logger.

Reply:
Ok, but we should use our logger and not another and another and do something different here and there and all ends up in a mess. So I would say lets please use our logger and do this in other projects as well.